spec/features/user_can_add_a_devtool_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'A user can add a new dev tool' do
+  context 'An authenticated user can add a dev tool' do
+    let!(:devtool1) {FactoryGirl.create(:devtool)}
+    before { visit new_devtool_path }
+    scenario 'The new page has a title, body and github text area' do
+      expect(page).to have_content("Title")
+      expect(page).to have_content("Body")
+      expect(page).to have_content("Github link")
+    end
+
```

Comment:
Again, the user should be filling out a form to add this dev tool, and then be redirected to that new dev tool's show page